examples/tweet_generator/tweet_generator.py

```diff
@@ -0,0 +1,195 @@
+import re
+import textwrap
+import os
+import sys
+from datetime import datetime
+from PIL import Image, ImageFont, ImageDraw, ImageOps
+from pincer import command, Client
+from pincer.commands import CommandArg, Description
+from pincer.objects import Message, Embed, MessageContext
+
+
+# you need to manually download the font files and put them into the folder
+# ./examples/tweet_generator/ to make the script works using this link:
+# https://fonts.google.com/share?selection.family=Noto%20Sans:wght@400;700
+if not all(
+        font in os.listdir()
+        for font in [
+            "NotoSans-Regular.ttf",
+            "NotoSans-Bold.ttf"
+        ]
+):
+    sys.exit()
```

Comment:
There should be a print statement here or an exception with more information on why it failed

Reply:
yes you're right !